pkgs/applications/audio/mopidy/youtube.nix

```diff
@@ -49,6 +29,25 @@ python3.pkgs.buildPythonApplication rec {
       --replace-fail '"youtube_dl_package": "youtube_dl",' '"youtube_dl_package": "yt_dlp",'
   '';
 
+  build-system = [
+    pythonPackages.setuptools
+  ];
+
+  dependencies = [
+    mopidy
+    pythonPackages.beautifulsoup4
+    pythonPackages.cachetools
+    pythonPackages.pykka
+    pythonPackages.requests
+    pythonPackages.ytmusicapi
+    pythonPackages.yt-dlp
+  ] ++ extraPkgs pkgs; # should we remove this? If we do, don't forget to also change the docs!
```

Comment:
This was originally added so that `youtube_dl` could be swapped out for `yt_dlp`.
Now, `yt_dlp` is patched in as the default value.
Should we still provide a way to add extra pkgs?